classes/Product.php

```diff
@@ -4093,18 +4093,7 @@ public function getIdProductAttributeMostExpensive()
      */
     public function getDefaultIdProductAttribute()
```

Comment:
@matks 

I think there is a proper condition:
https://github.com/PrestaShop/PrestaShop/blob/develop/classes/Product.php#L1082

Reply:
I'm sorry 😅 I'm still lost. I'm lost because the code of these 2 functions is quite hard to read, but if I'm confused, other people will be confused. That being said if you can decipher this code go ahead and review the PR 😅 but I can't review what I don't understand